<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <group ns="robot2">
        <param name="x_" value="1.095354"/>
        <param name="y_" value="1.488496"/>
        <param name="robot_prefix" value="robot2"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" />
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group> 
    <group ns="robot3">
        <param name="x_" value="-0.904977"/>
        <param name="y_" value="-0.032046"/>
        <param name="robot_prefix" value="robot3"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" output="screen"/>
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group>
    <group ns="robot4">
        <param name="x_" value="-0.916474"/>
        <param name="y_" value="1.504146"/>
        <param name="robot_prefix" value="robot4"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" />
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group> 
    <group ns="robot5">
        <param name="x_" value="1.063800"/>
        <param name="y_" value="-1.507870"/>
        <param name="robot_prefix" value="robot5"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" />
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group> 
    <group ns="robot6">
        <param name="x_" value="1.108737"/>
        <param name="y_" value="-0.051923 "/>
        <param name="robot_prefix" value="robot6"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" />
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group> 
    <group ns="robot7">
        <param name="x_" value="-0.879732"/>
        <param name="y_" value="-1.495313"/>
        <param name="robot_prefix" value="robot7"/>
        <node name="aol_controller" pkg="multi_controller" type="aol_controller.py" />
        <!-- <node name="target_detector" pkg="multi_controller" type="copDetector" /> -->
        <!-- <node name="mother" pkg="aol_stack" type="mother.py" output="screen"/> -->
    </group>  
</launch>
